projects/js-packages/publicize-components/src/components/share-buttons/copy-to-clipboard.tsx

```diff
@@ -0,0 +1,31 @@
+import { CopyToClipboard as CopyToClipboardBtn } from '@automattic/jetpack-components';
+import { useAnalytics } from '@automattic/jetpack-shared-extension-utils';
+import { useCallback } from '@wordpress/element';
+import styles from './styles.module.scss';
+import { ShareButtonProps } from './types';
+import { usePrepareUrl } from './usePrepareUrl';
+import type React from 'react';
+
+export const CopyToClipboard: React.FC< ShareButtonProps > = ( {
+	buttonStyle = 'icon',
+	buttonVariant,
+} ) => {
+	const prepareUrl = usePrepareUrl();
+	const { recordEvent } = useAnalytics();
+
+	const onCopy = useCallback( () => {
+		recordEvent( 'jetpack_social_share_button_clicked', { network: 'clipboard' } );
+	}, [ recordEvent ] );
+
+	const textToCopy = useCallback( () => prepareUrl( '{{text}}\n{{url}}', false ), [ prepareUrl ] );
```

Comment:
It was because that array of networks includes the list of social networks and all of them have a URL attached to them which will be opened in a window on click, but in case if Copy to Clipboard, it's not a social network and doesn't have a URL to open.